src/native-stack/views/NativeStackView.tsx

```diff
@@ -130,7 +130,11 @@ const MaybeNestedStack = ({
   if (isHeaderInModal) {
     return (
       <ScreenStack style={styles.container}>
-        <Screen enabled isNativeStack style={StyleSheet.absoluteFill}>
+        <Screen
+          enabled
+          isNativeStack
+          isLargeHeader={isLargeHeader}
```

Comment:
Nitpicking but `isLargeHeader` does not fit here for me. As the `Screen` is not a header, but it may have one, thus `hasLargeHeader` maybe? 

```suggestion
          isLargeHeader={isLargeHeader}
```

Reply:
Sure!

Reply:
Done in https://github.com/software-mansion/react-native-screens/pull/1915/commits/cf7dcb1b684a51d8f0799a3cc49342a676b6e658.